### PR TITLE
Add new endpoint build api

### PR DESCRIPTION
**Pull request by j0nathan33**
My last change for this month. Go ahead for next release after accept my pull Request

### CODE REVIEW

File: src/test/java/com/cdancy/bitbucket/rest/features/BuildApiMockTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Mock tests for the {@link BuildApi} class.
+ */
+@Test(groups = "unit", testName = "BranchApiMockTest")
```
Comment (cdancy) [inline src/test/java/com/cdancy/bitbucket/rest/features/BuildApiMockTest.java]: Lets change testName to `BuildApiMockTest`

File: src/main/java/com/cdancy/bitbucket/rest/BitbucketApi.java [hunk 2]
```diff
@@ -59,4 +60,7 @@
 
     @Delegate
     AdminApi adminApi();
+
+    @Delegate
+    BuildApi buildApi();
```
Comment (cdancy) [inline src/main/java/com/cdancy/bitbucket/rest/BitbucketApi.java]: Lets rename to `BuildStatusApi` to better align with this http path.

**Comment by cdancy** [comment]
@j0nathan33 looks good. One more minor comment to address and we should be good to go.

**Comment by j0nathan33** [comment]
I will need release for monday (may 1 2017)

**Comment by cdancy** [comment]
@j0nathan33 been a bit pre-occupied with our move to AWS/CBJ. I'll run the live tests now and merge if all is Ok then do another release. Stay tuned...

**Comment by cdancy** [comment]
Integration tests pass as expected. Thanks @j0nathan33 !

**Comment by cdancy** [comment]
@j0nathan33 v0.0.15 has been released. 

FYI: massaged your last PR a bit so your local branch may be off.

**Pull request merged**